htdocs/fourn/class/fournisseur.product.class.php

```diff
@@ -889,6 +894,18 @@ public function find_min_price_product_fournisseur($prodid, $qty = 0, $socid = 0
 							$fourn_unitprice_with_discount = $fourn_unitprice * (1 - $record["remise_percent"] / 100);
 						}
 					}
+
+                                        /* VSPR let us recalculate the objects values. Based on the Modul Extra Discounts adding 4 more discounts to supplier*/
+                                        global $action;
+                                        $hookmanager->initHooks(array($this->element . 'minprice'));
```

Comment:
inithook is a method called by the main page or cron launcher. 
Should not be done inside a low level method.

Reply:
I am not a real developer so I searched, but in fournisseur.product.class.php no other initHook was found. Is there a way to see the initialized hook at this position?